### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by bgavrilMS**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+A clear and concise description of what you expected to happen (or code).
+
+**Actual behavior**
+A clear and concise description of what happens, e.g. exception is thrown, UI freezes  
+
+**Possible Solution**
+<!--- Only if you have suggestions on a fix for the bug -->
+
+**Additional context/Screenshots**
+Add any other context about the problem here. If applicable, add screenshots to help explain.
```
Comment (jennyf19) [inline .github/ISSUE_TEMPLATE/bug_report.md]: What about asking for logs?

**Review by henrik-me — APPROVED** [review]
:shipit:

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (jennyf19) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Also, I find it interesting to know if the scenario or set-up has ever worked, or if this is the first time they've attempted a certain scenario...but maybe that's something for us to follow up on later w/customer.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Which Auth library and version are you using ?**
+Note that to get help, you need to run the latest preview or non-preview version
+<!-- E.g. ADAL 3.19.8, MSAL 1.1.4-preview -->
```
Comment (jmprieur) [inline .github/ISSUE_TEMPLATE/bug_report.md]: In the ADAL repo, we would not want issues for MSAL?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
```
Comment (jmprieur) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Which application type? and scenario
- Desktop 
  - interactive
  - WIA
  - U/P
  - Device code flow (browserless)
- Mobile 
- Web App 
  - authorization code
  - OBO
- Web API 
  - OBO

**Review by jmprieur — APPROVED** [review]
I've left a few suggestions

**Pull request merged**